### PR TITLE
[CI-3614] Don't send autocomplete requests with queries longer than 200 characters long

### PR DESCRIPTION
**Pull request by esezen**

### CODE REVIEW

File: src/hooks/useDebouncedFetchSections.ts [hunk 1]
```diff
@@ -93,7 +93,11 @@ const useDebouncedFetchSection = (
 
   useEffect(() => {
     (async () => {
-      if (debouncedSearchTerm.trim()) {
+      // Do not send a request if the encoded term is longer than 199 characters
+      if (
+        debouncedSearchTerm.trim() &&
+        encodeURIComponent(debouncedSearchTerm.trim()).length < 200
+      ) {
```
Comment (jjl014) [inline src/hooks/useDebouncedFetchSections.ts:100]: Should we save `debouncedSearchTerm.trim()` to a variable here?

**Review by jjl014 — APPROVED** [review]
Looks good from what I can tell! 😄 

Just left one minor comment.

File: src/hooks/useDebouncedFetchSections.ts (hunk 1 above)
Comment (esezen) [reply src/hooks/useDebouncedFetchSections.ts]: Optimization, I like it 🧠

**Review by jjl014 — APPROVED** [review]
LGTM! 🚀

**Pull request merged**